sql/src/main/java/io/crate/planner/Planner.java

```diff
@@ -144,7 +152,8 @@ public Planner(Settings settings,
                    LicenseService licenseService,
                    NumberOfShards numberOfShards,
                    TableCreator tableCreator,
-                   Schemas schemas) {
+                   Schemas schemas,
+                   UserManager userManager) {
```

Comment:
lgtm! 

one question, is there any reasoning why we inject `UserManager` in the `Planner` ctor, but not in `DependencyCarrier` because pretty many of other deps that are needed for plans are accessed via `DependencyCarrier`.

Reply:
The planner is instantiated manually inside the test framework and thus it would be required to create a `DependencyCarrier`  manually as well while just few things are used.